### ISSUE TITLE
Feature request: Support for Node 20

### ISSUE THREAD

**Issue by Firefds** [DataDog/serverless-plugin-datadog#447]
Node.js 20.x runtime now is available in AWS Lambda - https://aws.amazon.com/blogs/compute/node-js-20-x-runtime-now-available-in-aws-lambda/

Would it be possible to add Node 20 support?  Our test deployments are showing warnings 
```
Unable to add Lambda Layers to function xxx with runtime nodejs20.x
```

Edit: this [PR](https://github.com/DataDog/datadog-lambda-js/pull/437) is waiting to be approved.

**Comment by astuyve** [comment]
Hi @Firefds - thanks! Yes, this is in progress and should be released soon.
Thanks!

**Comment by Firefds** [comment]
Hi @astuyve! I opened a PR for this - https://github.com/DataDog/serverless-plugin-datadog/pull/448

**Comment by astuyve** [comment]
[released in 5.53](https://github.com/DataDog/serverless-plugin-datadog/releases/tag/v5.53.0)

Thanks folks!!
